src/views/_proxied-dot-io/product-docs-landing/partials/use-case-card/types.ts

```diff
@@ -0,0 +1,10 @@
+import { Products } from '@hashicorp/platform-product-meta'
+export interface UseCaseCardProps {
```

Comment:
nit suggestion

```suggestion
import { Products } from '@hashicorp/platform-product-meta'

export interface UseCaseCardProps {
```

Reply:
applied this! (well, renamed the file first - should have thought through that and committed the suggestion first 😅 )